cogs/StaffAppsBackoffice.py

```diff
@@ -0,0 +1,966 @@
+        super().__init__()
+        self.author = author
+        self.data = data
+        self.cur_page = 1
+        self.max_page = len(data) // 10 + 1
+        self.da = da  # d for denied, a for accepted
+
+    async def repopulate(self, interaction: discord.Interaction) -> None:
+        """
+        Repopulates the view with the next page of data after a button press.
+        """
+        title = f"{'Denied' if self.da == 'd' else 'Accepted'} Applications page {self.cur_page}/{self.max_page}"
+        field_title = "Application ID, Discord Name, Original Submission Time"
+        field_content = ""
+        for row in self.data[self.cur_page * 10 - 10 : self.cur_page * 10]:
+            field_content += (
+                f"{row[0]}, {row[1]}, {datetime.fromtimestamp(int(row[2]))}\n"
+            )
+        fields = [[field_title, field_content, False]]
+        embed = EmbedBuilder(
+            title=title,
+            description="",
+            fields=fields,
+            color=0x9D0006 if self.da == "d" else 0x80C545,
+        )  # soft red or hwh green
+        await interaction.response.edit_message(embed=embed.build(), view=self)
+
+    @discord.ui.button(
+        label="",
+        style=discord.ButtonStyle.primary,
+        emoji="⬅️",
+        disabled=True,
+    )
+    async def button_callback(
+        self,
+        button: discord.ui.Button,
+        interaction: discord.Interaction,
+    ) -> None:
+        self.children[1].disabled = False
+        self.cur_page -= 1
+        if self.cur_page == 1:
+            button.disabled = True
+        await self.repopulate(interaction)
+
+    @discord.ui.button(
+        label="",
+        style=discord.ButtonStyle.primary,
+        emoji="➡️",
+        disabled=False,
+    )
+    async def button2_callback(
+        self,
+        button: discord.ui.Button,
+        interaction: discord.Interaction,
+    ) -> None:
+        self.children[0].disabled = False
+        self.cur_page += 1
+        if self.cur_page == self.max_page:
+            button.disabled = True
+        await self.repopulate(interaction)
+
+
+class staffAppsSeeSpam(discord.ui.View):
+    def __init__(self, author: discord.Member, data: list) -> None:
+        """
+        A discord view for the spam command
+        """
+        super().__init__()
+        self.author = author
+        self.data = data
+        self.cur_page = 1
+        self.max_page = len(data)
+
+    async def repopulate(self, interaction: discord.Interaction) -> None:
+        self.db = sqlite3.connect("util/database.sqlite")
+        self.cursor = self.db.cursor()
+        first_user_data = self.cursor.execute(
+            "select * from application where uid = ?",
+            (self.data[self.cur_page - 1][0],),
+        ).fetchall()
+        self.db.close()
+        count = len(first_user_data)
+        name = first_user_data[0][3]
+        most_recent = datetime.fromtimestamp(
```

Comment:
datetime.fromtimestamp() takes an EPOCH timestamp as argument, EPOCH timestamps are the time in seconds elapsed since 00:00:00 UTC on 1 January 1970. This object is already timezone-aware at the time of creation.